ui/templates/robots.txt

```diff
@@ -0,0 +1,5 @@
+User-agent: *
+Disallow: /dashboard/
+Disallow: /learner/
+Disallow: /learners/
+Disallow: /api/
```

Comment:
i dont think so apis are useful for crowlers or rebots? because they are session oriented. No one can use it without login.
@pdpinch @giocalitri